code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -4405,6 +4405,33 @@
 
 	M.nutrition += nutriment_factor
 
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		switch(volume)
+			if(1 to 15)
+				if(prob(5))
+					to_chat(H,"<span class='warning'>You burp.</span>")
+					holder.remove_reagent(src.id, 0.1 * FOOD_METABOLISM)
+			if(15 to 200)
```

Comment:
I think you meant 100 not 200, a few lines below you do `if(100 to INFINITY)`

Reply:
I won't really change anything but if your oce wants it